pandas/tests/tslibs/test_tslib.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from datetime import timedelta
+
+import numpy as np
+
+from pandas._libs.tslib import array_to_datetime
+from pandas import Timestamp
+
+import pandas.util.testing as tm
+
+
+class TestArrayToDatetime(object):
```

Comment:
hmm this should be in ``pandas/tests/indexes/datetimes/test_tools.py``. I know you are ceating new locations is fine. but would rather move all of tools testing there (in a new PR).